MovingBlocks/Terasology
# Rebased gltf support

### Contains

Completes PR #3785 by rebasing the engine to support BiomesAPI.

### How to test

Create a world with the BiomesAPI module activated
To check gltf support run command: spawnPrefab "engine:cubesm"

### Outstanding before merging

- [ ] Convert old models from .md5 to .gltf
